### PR TITLE
A lot of stuff

### PR DESCRIPTION
**Pull request by marcos-mendez**
Fixing also this #1011 and  other stuff with pre-commit

### CODE REVIEW

**Comment by marcos-mendez** [comment]
@marcelsavegnago and @rvalyi if you could help me review this i'll appretiate

File: fieldservice_equipment_qrcode/reports/fieldservice_equipment_report_template.xml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+                            <p><b>EQUIPO:</b> <t t-esc="o.name" /></p>
+                            <p><b>FABRICANTE:</b> <t
+                                t-esc="o.product_id.product_brand_id.name"
+                            /></p>
+                            <p><b>MODELO:</b> <t
+                                t-esc="o.product_id.categ_id.name"
+                            /></p>
+                            <p><b>N#:</b> <t t-esc="o.lot_id.name" /></p>
+                            <p><b>LOCALIZACAO ORIGINAL:</b><br /> <t
+                                t-esc="o.current_location_id.name"
```
Comment (rvalyi) [inline fieldservice_equipment_qrcode/reports/fieldservice_equipment_report_template.xml:44]: Reports should be in English and translated only using the i18n files.

**Review by rvalyi — CHANGES_REQUESTED** [review]
It seems you are changing/adding several modules. I would say start with basic things and submit more modules for review as you have some PRs getting merged... 
In general, It's unlikely people spend time reviewing hundreds of lines of third party code from several modules if you don't convince them first it's worth their time with smaller steps...

**Comment by marcos-mendez** [comment]
I dont know how to push only one module
can you teach me ?

**Comment by marcos-mendez** [comment]
@rvalyi

**Comment by marcelsavegnago** [comment]
> @marcelsavegnago and @rvalyi if you could help me review this i'll appretiate

@marcos-mendez Unfortunately, as it stands, I can't help with the review. I suggest you do the PRs by module or feature/improvement. Also, from what I've seen, the commits need a review/cleanup.

I suggest you make the PRs similar to what your developer did for your fork. If in doubt, check the repository's PR history and base it on the way other collaborators do.

**Comment by marcelsavegnago** [comment]
Example of a commit that is not good. I recommend making commits separated by module as this makes porting commits forward or backwards easier.

https://github.com/OCA/field-service/pull/1022/commits/89e896dac8eb7925d0496c29342b8278d810141a

**Pull request closed without merging**